librz/core/cfile.c

```diff
@@ -659,15 +659,17 @@ static int rz_core_file_do_load_for_debug(RzCore *r, ut64 baseaddr, const char *
 	}
 #endif
 	int fd = cf ? cf->fd : -1;
+
 	RzBinOptions opt;
-	rz_bin_options_init(&opt, fd, baseaddr, UT64_MAX, false, false);
+	rz_bin_options_init(&opt, fd, baseaddr, UT64_MAX, false, rz_config_get_i(r->config, "bin.rawstr"));
+	opt.obj_opts.elf_load_sections = rz_config_get_b(r->config, "elf.load.sections");
+	opt.obj_opts.elf_checks_sections = rz_config_get_b(r->config, "elf.checks.sections");
+	opt.obj_opts.elf_checks_segments = rz_config_get_b(r->config, "elf.checks.segments");
```

Comment:
This seems different from previous behaviour. Was it tested?

Reply:
Yes, in linux64/*. I believe.